DER/DerivativesContracts/SwapsIndividuals.rdf

```diff
@@ -128,8 +128,7 @@
 	</owl:NamedIndividual>
 	
 	<owl:NamedIndividual rdf:about="&fibo-der-drc-swpind;BSDRHeadquartersAddress">
-		<rdf:type rdf:resource="&fibo-be-le-fbo;RegisteredAddress"/>
-		<rdf:type rdf:resource="&fibo-fbc-fct-breg;RegistrationAddress"/>
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
```

Comment:
Why is this not of type ConventionalStreetAddress?

Reply:
It could have been - I did the replacement for registration address first, which could also be a Post Office Box address, which is not a conventional street address necessarily.